docs/source/development_guide/ci.rst

```diff
@@ -10,28 +10,89 @@ it implies that the changes introduced by a commit does not introduce any errors
 When creating a PR to master branch or when pushing to master branch, around 35 CI
 tests are run from the following workflows.
 
-#. Tests for DeepChem Core - The jobs are defined in the ``.github/workflows/main.yml`` file. The following jobs are performed in this workflow:
```

Comment:
Some of the context present in the deleted comments is not present in the updated comments. Make sure that we don't lose context because of updates

Reply:
Added more context